core/mocks/trezortranslate_keys.pyi

```diff
@@ -601,15 +601,15 @@ class TR:
     reset__threshold_info: str = "= minimum number of unique word lists used for recovery."
     reset__title_backup_is_done: str = "BACKUP IS DONE"
     reset__title_create_wallet: str = "CREATE WALLET"
-    reset__title_create_wallet_shamir: str = "CREATE WALLET (SHAMIR)"
+    reset__title_create_wallet_shamir: str = "CREATE WALLET (MULTI-SHARE)"
```

Comment:
This would be visible also when user is creating 1 of 1 right?

During the creation of 1of1 shamir, there cannot be a term multi-share...

Reply:
Good point. I'll use something else in that case. What should it be? "single share"?

Reply:
No, we now cannot distinguish which type of backup user would do (single-share or multi-share). So for now on deviceReset let's use just Create wallet as in bip39 flow.